v1.2 made edits for 1.2 release (#130)
mulesoft/docs-service-mesh: modules/ROOT/pages/index.adoc

== Anypoint Service Mesh Architecture

> Anypoint Service Mesh is an independent architecture layer, which is encapsulated in a Kubernetes or Red Hat OpenShit cluster. In the Anypoint Service Mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job.  

Reviewer: ```suggestion
Anypoint Service Mesh is an independent architecture layer encapsulated in a Kubernetes or Red Hat OpenSht cluster. In the Anypoint Service Mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job.  
```
```suggestion
Anypoint Service Mesh is an independent architecture layer, which is encapsulated in a Kubernetes or Red Hat OpenShit cluster. In the Anypoint Service Mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job.  
```

Author: Done! Thanks!

> Anypoint Service Mesh is an independent architecture layer, which is encapsulated in a Kubernetes or Red Hat OpenShit cluster. In the Anypoint Service Mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job.  

Reviewer 2: ```suggestion
Anypoint Service Mesh is an independent architecture layer, which is encapsulated in a Kubernetes or Red Hat OpenShift cluster. In the Anypoint Service Mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job.  
```

Author: Done! Thanks!

> Anypoint Service Mesh is an independent architecture layer, which is encapsulated in a Kubernetes Cluster (optionally with Red Hat OpenShit installed for additional security). In the Anypoint Service Mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job.  

Reviewer 2: ```suggestion
Anypoint Service Mesh is an independent architecture layer, which is encapsulated in a Kubernetes or Red Hat OpenShit Cluster. In the Anypoint Service Mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job.  
```

Author: Done! Thanks!
